File: 20240601070 SE226 LAB1/lab1 - c++/coordinate_geometry.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main() {
int x1{};
int y1{};
int x2{};
int y2{};

cout << "Enter x coordinate for first point: "<< endl;
cin >> x1;
cout << "Enter y coordinate for first point: "<< endl;
cin >> y1;
cout << "Enter x coordinate for second point: "<< endl;
cin >> x2;
cout << "Enter y coordinate for second point: "<< endl;
cin >> y2;

double distance = pow((pow((x2-x1),2) + pow((y2-y1),2)),0.5);
cout << distance << endl;

return 0;
}

File: 20240601070 SE226 LAB1/lab1 - python/coordinate_geometry.py
x1 = int(input("Enter x coordinate for first point"))
y1 = int(input("Enter y coordinate for first point"))

x2 = int(input("Enter x coordinate for second point"))
y2 = int(input("Enter y coordinate for second point"))

distance = ((x2 - x1)**2 + (y2 - y1)**2)**0.5
print(distance)